migtests/scripts/run-test.sh

```diff
@@ -38,6 +38,9 @@ main() {
 	step "Initialise source database."
 	./init-db
 
+	step "Grant source database user permissions"
+	grant_user_permission_${SOURCE_DB_TYPE} ${SOURCE_DB_NAME}
```

Comment:
This will break the automation for oracle.

Reply:
What can we do about this?

Reply:
For now, at least define an empty function for oracle.

Also, let's not use the shell magic to call source db specific function. Instead, define a function `grant_permissions` that takes `SOURCE_DB_TYPE` and `SOURCE_DB_NAME` as arguments and use a case statement to call source db type specific functions.

Reply:
Done